src/master-playlist-controller.js

```diff
@@ -455,6 +455,7 @@ export class MasterPlaylistController extends videojs.EventTarget {
       videojs.log.warn('Problem encountered with the current subtitle track. Switching' +
                        ' back to default.');
       this.subtitleSegmentLoader_.abort();
+      this.subtitlePlaylistLoader_ = null;
```

Comment:
I was trying to match alt audio, but I think you're right, it isn't needed here.